src/models/methods/ledger.ts

```diff
@@ -34,7 +34,10 @@ interface Ledger {
 
 interface LedgerQueueData {
     account: string
-    tx: any // TODO: Retype this once we have transaction types (Also include tx_blob as possible type: https://xrpl.org/ledger.html)
+    // TODO: Retype tx once we have transaction types 
+    // TODO: Also include tx_blob as possible type: https://xrpl.org/ledger.html
+    // TODO: Also handle the special case where 'owner_funds: string' is a field of OfferCreate sometimes - https://xrpl.org/ledger.html#response-format
+    tx: any 
```

Comment:
Doesn't hurt to have all these TODOs listed, but a) it should just be under one TODO (so if you're just counting all the TODOs in the code these don't all pop up as separate), and b) IMO re-typing implies you take a look at what the typing should be later